pkg/server/exec/exec.go

```diff
@@ -56,10 +57,14 @@ func (ops *ExecOperations) RunCommand(ctx context.Context, user *database.User,
 		return nil, errChan
 	}
 
+	imgs := &spec.SlugImages{
+		Runner: ops.defaults.RunnerImage,
+		Store:  ops.defaults.StoreImage,
+	}
```

Comment:
I'm a bit curious, you created this struct to avoid another argument on `spec` functions?

Reply:
Yep, the  argument list is gigantic.

Reply:
hm, ok 👍 for a next refactor maybe we could create a struct with all args, something like `options{}`